### PR TITLE
Deprecated mattermost version CLI command

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost-server/pull/19364
- Updated the ``mattermost version`` CLI command note to deprecate the command in favor of ``mattermost db migrate``.

### CODE REVIEW

**Comment by cwarnermm** [comment]
@isacikgoz - Given that ``mattermost version`` is being deprecated, can I get your help updating a few examples of ``mattermost version`` usage on this CLI docs page, please? Instead of ``mattermost version``, what Mattermost CLI command, if any, would you recommend as a replacement? (Or would you recommend taking a different path altogether with these updates?)
- https://docs.mattermost.com/manage/command-line-tools.html#use-the-cli
- https://docs.mattermost.com/manage/command-line-tools.html#use-the-cli-on-gitlab-omnibus

**Review by isacikgoz — COMMENTED** [review]
@cwarnermm `mmctl system version` returns the build number only. I think it won't be same as `mattermost version`. Having said that, we still ship `mattermost version`, the only change is it isn't interact with the database anymore. I tried to simplify the wording like below but let me know what you think.

**Comment by cwarnermm** [comment]
Thank you @isacikgoz! The fact that mmctl system version only returns a build number is worth clarifying in the docs too. Given that we have a couple of instances on that page where we use `mattermost version` as an example, what would you recommend as a reasonable alternative example instead?

**Comment by isacikgoz** [comment]
@cwarnermm I think the examples in the `manage/command-line-tools.html` path are just fine. Maybe we can omit the `--config` flag in the GitLab Omnibus example and it's corresponding value as the flag is redundant and not being used anymore.

The mmctl documentation for `mmctl system version` is accurate but it can be elaborated with an example output anyway.

**Review by isacikgoz — APPROVED** [review]
Awesome @cwarnermm 🎉

**Review by justinegeffen — APPROVED** [review]
